src/libs/utils.js

```diff
@@ -95,6 +99,11 @@ export const makeStandardDate = dateString => dateFormat.format(new Date(dateStr
 
 export const makeCompleteDate = dateString => completeDateFormat.format(new Date(dateString))
 
+export const makeCompleteDateParts = dateString => {
+  const date = new Date(dateString)
+  return _.map(part => part.format(date), completeDateFormatParts)
```

Comment:
Not sure you're getting a lot from pulling out the construction, so perhaps:
```suggestion
  return _.map(part => part.format(new Date(dateString)), completeDateFormatParts)
```